### PR TITLE
[WIP] translation chapter 1 es to en

### PR DESCRIPTION
**Pull request by stephien**

### CODE REVIEW

**Comment by luisan00** [comment]
@stephien , is thiis a WIP or a Ready for review PR?

**Comment by stephien** [comment]
@luisan00 this is a WIP, forgot to add the tag!

**Comment by luisan00** [comment]
ready @stephien :+1:

File: documentation/en/chapter_1.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Locha Mesh main objective is the implementation of the routing protocol AODVv2 to enable a resilient communication system, composed of the following:
+
+1. Choosing the most suitable routing protocol for the packets transmision between nodes.
+2. Select the proper hardware for the execution of the software and storing the necessary data. 
+3. Implementation of the AODVv2 protocol in an embedded system, for this case [RIOT-OS](https://www.riot-os.org/).
+4. Verify the performance of the AODVv2 protocol and contrast it with what was previously planned. 
+5. Development of the driver needed for the integration with the OS and the hardware. 
+6. Calculate and set constant values for the network operation.
+
+Our secondary objectives are:
```
Comment (gustavosinbandera1) [inline documentation/en/chapter_1.md:45]:
```suggestion
Our secondary challenges are:
```

File: documentation/en/chapter_1.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- Theoretical researh for ad-hoc wireless networks and routing protocols.
```
Comment (gustavosinbandera1) [inline documentation/en/chapter_1.md:47]:
```suggestion
- Theoretical research for ad-hoc wireless networks and routing protocols.
```

File: documentation/en/chapter_1.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+## 1. Motivation
+
+Even though telecom tecnology has taken a big leap in terms of development, it becomes evident it's own susceptibility. We're not talking about tecnology by itself but about people, those are the ones vulnerable to the side effects of communication isolation resulting from centralization, and regulation of communication standards worldwide.
+
+Since the invention of wireless networks in 1970s, it's growth and demand in the last decade has been exponential. Specifically, there are two paths in this subject that attend to the underlying architecture of the network: wireless networking infrastructure mode and ad-hoc mode networks.
```
[the remark this answers is not in the document]
Comment (stephien) [reply documentation/en/chapter_1.md:5]: I like the 'perspectives' addition, but i will commit changes in this line manually.

File: documentation/en/chapter_1.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+As an example for wireless networking infrastructure mode we can take a look to cellular networks, in these networks the user can connect wireless to the main network as long as it's inside the coverage range of the network, this is how it can serve a large number of users inside an extense geographic area by using a limited frequency spectrum. The service provided is high quality and is often compared to the wired telephone systems. In this network, the routers or base stations, are in fixed-locations and wired, the mobile units connect to it and communicate through it. If this mobile unit goes further than the range of its base a handoff to another base its produced.
```
[the remark this answers is not in the document]
Comment (stephien) [reply documentation/en/chapter_1.md:7]: What's the issue here?

File: documentation/en/chapter_1.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+The popularity of the wireless networks started back in the 1980s with the deployment of analog 1G phones limited to one country due to the different standards established by geographic situation. These conditions encourage the development of the 2G for the next decade, adding fax, data service and SMS, but it wasn't adopted as a worldwide standard. This generation was recently extended to 2.5G improving the support of low and high speed data transmition. The transition to 3G determined the adoption of a globalized standard adding roaming, and upgrading the size of the bandwith considerably. In spite of these improvements, 3G presented some difficulties in its deployment and also in the delivering of its services due to limitations on the architecture of the network, which resulted in the creation of the 4G network based on VoIP, relaying all of the services through IP.
+
+Another relevant example of wireless networking infrastructure mode are the WLAN (Wireless Local Area Network) commonly known as Wi-Fi and Bluetooth which are vastly used in millions of daily used product that connect between devices or serve as bridge to internet.
```
[the remark this answers is not in the document]
Comment (stephien) [reply documentation/en/chapter_1.md:11]: (Y)

**Pull request closed without merging**